core/src/main/java/com/novoda/landingstrip/animation/FastOutSlowInInterpolator.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.landingstrip.animation;
+
+class FastOutSlowInInterpolator extends LookupTableInterpolator {
```

Comment:
have been trying to find the design lib license, might have to plonk in the aosp one